scripts/irods/test/test_ireg.py

```diff
@@ -318,8 +318,8 @@ def test_ireg_replica_which_does_not_exist_on_connected_host__issue_4206(self):
         """Test registering a file as a replica which does not exist on this host."""
         resource = 'reghere'
 
-        old_path_to_file = os.path.join('/var', 'lib', 'irods', 'VERSION.json.dist')
-        new_path_to_file = os.path.join('/var', 'lib', 'irods', 'VERSION.json.dist.new')
+        old_path_to_file = os.path.join('/var', 'lib', 'irods', 'version.json.dist')
+        new_path_to_file = os.path.join('/var', 'lib', 'irods', 'version.json.dist.new')
```

Comment:
Anybody familiar with this test and what it's doing?

Reply:
This change is fine. The version file's use here isn't special. Based on the test, any file could have been used.

Reply:
Right, this test just picked something that was guaranteed to be on the other computer in a test-configured environment without yet having a supported capability for making files on the other machines in a topology. I suppose you could do some `scp`/`ssh` nonsense but then we'd have to add more dependencies, potentially.